x-pack/plugins/session_view/common/types/session_view/index.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface OptionsField {
```

Comment:
Should these not live in session_view_options? Seems like they are only used in that component.

Reply:
nevermind I see it's used below.